programs/vote/src/vote_state/mod.rs

```diff
@@ -54,6 +54,10 @@ impl Vote {
             timestamp: None,
         }
     }
+
+    pub fn last_voted_slot(&self) -> Option<Slot> {
+        self.slots.last().copied()
```

Comment:
I need to start being more mindful to d this over `.cloned()` when possible, good reminder!